### PR TITLE
DOCSP-45175: Connect to MongoDB

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-45175

### Staging Links

<li><a href=https://deploy-preview-95--docs-ruby.netlify.app/get-started/connect-to-mongodb>get-started/connect-to-mongodb</a></li><li><a href=https://deploy-preview-95--docs-ruby.netlify.app/get-started/next-steps>get-started/next-steps</a></li><li><a href=https://deploy-preview-95--docs-ruby.netlify.app/get-started>get-started</a></li><li><a href=https://deploy-preview-95--docs-ruby.netlify.app/get-started/download-and-install>get-started/download-and-install</a></li>

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/get-started/connect-to-mongodb.txt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+.. procedure::
+   :style: connected
+
+   .. step:: Edit your {+language+} application file
+
+      Navigate to your ``quickstart.rb`` file in the ``ruby-quickstart``
+      directory. Copy and paste the following code under the Bundler
+      code from the :ref:`ruby-quick-start-download-and-install` step 
+      of this tutorial to query the ``movies`` collection in the ``sample_mflix``
+      database:
```
Comment (jordan-smith721) [inline source/get-started/connect-to-mongodb.txt:23]: S: Split this last sentence since it's a bit long and hard to follow
```suggestion
      directory. Copy and paste the following code below the Bundler
      code from the :ref:`ruby-quick-start-download-and-install` step 
      of this tutorial. This code connects to MongoDB and queries the 
      ``movies`` collection in the ``sample_mflix`` database.
```

File: source/get-started/connect-to-mongodb.txt [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+   .. step:: Assign the connection string
+
+      Replace the ``<connection string>`` placeholder with the 
+      connection string that you copied from the :ref:`ruby-get-started-connection-string`
+      step of this tutorial.
+
+   .. step:: Run your {+language+} application
+
+      In your project directory, run the following shell command to run the application:
```
Comment (jordan-smith721) [inline source/get-started/connect-to-mongodb.txt:39]: S: Be specific about where to run this from since the first step had them navigate away from the directory
```suggestion
      Navigate back to your ``ruby-quickstart`` directory and run the following shell command to run the application:
```

**Review by jordan-smith721 — APPROVED** [review]
A couple suggestions but otherwise LGTM!

File: source/includes/get-started/quickstart.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  movie = movies.find(query).first
+
+  # Prints the movie document
+  puts movie
+
+  client.close
+
+rescue => e
+  raise "Unable to find the document due to the following error: #{e}"
+end
```
Comment (comandeo-mongo) [inline source/includes/get-started/quickstart.rb:30]:
```suggestion
ensure
  client&.close
end
```

I do not think we need to catch errors here only to print them - uncatched errors will be printed to stderr anyways. Instead, we may put closing the client into `ensure`, this is a good practice. We also use `&` in the `client&.close` because `client` can be `nil` theoretically (if client creation failed).

**Review by comandeo-mongo — CHANGES_REQUESTED** [review]

**Review by comandeo-mongo — APPROVED** [review]
